apps/sr-frontend/src/app/Character.tsx

```diff
@@ -55,6 +60,114 @@ export default function Character() {
+                      onClick={() => charReducer({ eidolon })}
+                    >
+                      Eidolon Lv. {eidolon}
+                    </MenuItem>
+                  ))}
+                </DropdownButton>
+              </Grid>
+            </Grid>
+            <Accordion>
+              <AccordionSummary expandIcon={<ExpandMore />}>
+                Traces
+              </AccordionSummary>
+              <AccordionDetails>
+                <Stack spacing={1}>
+                  <Typography variant="h6">Abilites</Typography>
+                  <Divider />
+                  <Grid container gap={1}>
+                    <Grid item>
+                      <AbilityDropdown
+                        characterKey={characterKey}
+                        abilityKey="basic"
+                      />
+                    </Grid>
+                    <Grid item>
+                      <AbilityDropdown
+                        characterKey={characterKey}
+                        abilityKey="skill"
+                      />
+                    </Grid>
+                    <Grid item>
+                      <AbilityDropdown
+                        characterKey={characterKey}
+                        abilityKey="ult"
+                      />
+                    </Grid>
+                    <Grid item>
+                      <AbilityDropdown
+                        characterKey={characterKey}
+                        abilityKey="talent"
+                      />
+                    </Grid>
+                  </Grid>
+                  <Typography variant="h6">Bonus Abilities</Typography>
+                  <Divider />
+                  <Grid container gap={1}>
+                    {Object.entries(character?.bonusAbilities ?? {}).map(
+                      ([bonusAbility, enabled]) => (
+                        <Grid item key={bonusAbility}>
+                          <Button
+                            color="primary"
+                            onClick={() => {
+                              charReducer({
+                                bonusAbilities: {
+                                  ...character?.bonusAbilities,
+                                  [bonusAbility]: !enabled,
+                                },
+                              })
+                            }}
+                            startIcon={
+                              enabled ? <CheckBox /> : <CheckBoxOutlineBlank />
+                            }
+                          >
+                            {bonusAbility}
+                          </Button>
+                        </Grid>
+                      )
+                    )}
+                  </Grid>
+                  <Typography variant="h6">Stat Boosts</Typography>
+                  <Divider />
+                  <Grid container gap={1}>
+                    {Object.entries(character?.statBoosts ?? {}).map(
+                      ([statBoost, enabled]) => (
```

Comment:
looks like statboosts arent working on the calc side. nothing for you to fix, this is just a reminder for myself.